lib/chroma-docker-image-ecr-deployment-cdk-stack.ts

```diff
@@ -20,7 +20,7 @@ export class ChromaDockerImageEcrDeploymentCdkStack extends cdk.Stack {
         ecrRepository.addLifecycleRule({ maxImageAge: cdk.Duration.days(7), rulePriority: 1, tagStatus: ecr.TagStatus.UNTAGGED }); // delete images older than 7 days
         ecrRepository.addLifecycleRule({ maxImageCount: 4, rulePriority: 2, tagStatus: ecr.TagStatus.ANY }); // keep last 4 images
 
-        const deployImageVersions = [props.imageVersion, LATEST_IMAGE_VERSION];
+        const deployImageVersions = props.imageVersion === LATEST_IMAGE_VERSION? [props.imageVersion] : [props.imageVersion, LATEST_IMAGE_VERSION];
```

Comment:
**Suggestion:** To improve readability and maintainability, consider extracting the logic for determining `deployImageVersions` into a separate function.
> 🥝  **Medium Priority** 
 > [category: **`enhancement`**, importance: **`7`**]
> **Reason:** Extracting the logic into a separate function enhances readability and maintainability, making the code cleaner and more modular.
 

```suggestion
        const getDeployImageVersions = (imageVersion: string) => {
            return imageVersion === LATEST_IMAGE_VERSION ? [imageVersion] : [imageVersion, LATEST_IMAGE_VERSION];
        };
        const deployImageVersions = getDeployImageVersions(props.imageVersion);
```